Utilities/bootstrap

```diff
@@ -1183,13 +1235,17 @@ def main():
     # We need to embed an RPATH so swift-{build,test} can find the core
     # libraries.
     if args.stdlib_rpath:
+        assert(not platform.system() == 'Windows')
         embed_rpath = args.stdlib_rpath
     else:
-        if platform.system() == 'Linux':
-            embed_rpath = "$ORIGIN/../lib/swift/linux"
-        else:
+        if platform.system() == 'Darwin':
             embed_rpath = "@executable_path/../lib/swift/macosx"
-    build_flags.extend(["-Xlinker", "-rpath", "-Xlinker", embed_rpath])
+        elif platform.system() == 'Linux':
+            embed_rpath = "$ORIGIN/../lib/swift/linux"
+        elif platform.system() == 'Windows':
+            pass
+    if platform.system() == 'Darwin' or platform.system() == 'Linux':
```

Comment:
How about just:

    if embed_rpath:

so that this doesn't need to be kept in sync with the if-else above if additional platforms are added.